mainmatter/background_and_related_work/short_history/index.tex

```diff
@@ -98,75 +143,178 @@ \subsection{The Rise of Graphics \acs{API}s}
+This meant that the work done by vertex, tessellation, geometry and fragment shaders began to be performed by the same units on the device.   
+Having now reached the modern day, we will discuss the current state of Direct3D and OpenGL in the following section. 
+
+
+\subsection{State of Modern APIs}
+As has been the case previously, today the development of graphics \glspl{API} and the underlying driver libraries is done be separate companies, such as AMD or Nvidia.
+Often one company designs the \gls{API}, and hardware manufacturers can then decide to support the \gls{API} on their \glspl{GPU}.
+To add support for an \gls{API} on a specific \gls{GPU}, the manufacturer includes a library implementation as part of the device driver.
+Usually the driver must pass some tests and requirements, for the \gls{API} to be officially supported on the device.   
+In the case of modern Direct3D, the underlying driver is partially implemented by Microsoft in a closed common runtime, which communicates with the hardware vendors actual \gls{GPU} driver through the Direct3D \gls{DDI}\cite{dxDDI}.
+On the other hand, OpenGL is designed by several hardware and software vendors in cooperation, and requires that manufacturers write their own library from scratch. 
+
+
+A benefit of OpenGL is that manufacturers may provide extensions to the \gls{API} to give developers access to new hardware features \cite{openGLExtensions}. 
+Although this makes it more difficult to make code portable.
+With Direct3D, new hardware features are only provided to developers, when a new version of the \gls{API} is released by Microsoft. 
+One of the big differences between the two \glspl{API} is the extent of their platform support.
+Direct3D 11, being one of the latest releases from 2009, is supported on PCs running the Windows \gls{OS}, smartphones running the Windows Phone \gls{OS} as well as on Microsoft’s Xbox 360 and Xbox One.
+OpenGL, with OpenGL 4 being the latest major release from 2010, is considered cross platform.
+If a driver has been written, there is support for it on both Windows and Unix-based systems like macOS, iOS, Linux and Android. 
+
+
+Getting a clear picture of the market share of the two \glspl{API} is difficult.
+Though it seems that for many years Direct3D has had an upper hand.
+The people behind the popular Unity game engine collect system specifications on Unity users, as to help developers figure out what system configurations their games should support.
+Looking at the statistics from March 2017, only about 1\% of users use a non-Windows \gls{OS} \cite{unityStats}.
+Similar statistics are collected by Valve Corporation from users of their digital distribution platform, Steam.
+This shows a similar 3\% statistic for non-windows users\cite{steamStats}, implying that a large portion of gamers play on systems with some version of Direct3D enabled.
+
+
+This seems to indicate that Direct3D is a defacto-standard for graphics in games. 
```

Comment:
Bold claim, doesn't it only mean that Direct3D is widely available rather than a standard?